CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 11.2.0 (XX.XX.2021)
+
+### Features
+
+-   **terra-form**
```

Comment:
Könnte man hier vor der Auflistung generell noch einen Satz hinzufügen, der grob sagt, was passiert ist? Also sowas wie "For some form types, deprecated variants have been replaced as follows: ..... Oder ist das eher unüblich?

Reply:
Finde ich gut. Könnten wir dann für den eigentlichen Release noch anpassen! 😄

Reply:
Finde ich gut 👍

Reply:
Hab deinen Vorschlag mal umgesetzt 😄

Reply:
👍